packages/rn-tester/RNTester/NativeExampleViews/FlexibleSizeExampleView.mm

```diff
@@ -47,7 +47,7 @@ - (instancetype)initWithFrame:(CGRect)frame
     AppDelegate *appDelegate = (AppDelegate *)[[UIApplication sharedApplication] delegate];
 
     _resizableRootView =
-        (RCTRootView *)[appDelegate.rootViewFactory viewWithModuleName:@"RootViewSizeFlexibilityExampleApp"];
+        (RCTRootView *)[appDelegate.reactNativeFactory.rootViewFactory viewWithModuleName:@"RootViewSizeFlexibilityExampleApp"];
```

Comment:
I wonder if we can find a way not to reach for the AppDelegate... 🤔 but probably this is a problem for another time.

Reply:
Probably users could implement their own singelton wrapper for the factory? Not sure if there is other way to handle it. Making it a singelton by default isn't the best option as users may want to spawn multiple instances.